setup.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+"""Setup script."""
+
+# Copyright (c) 2020 SUSE LLC
+#
+# This file is part of mashhouse. mashhouse provides an api and
```

Comment:
The file is no longer part of masshouse, it is now part of suse-cloud-image-name-parser